synthpop.proto

```diff
@@ -121,6 +132,32 @@ enum NSSEC5 {
     ROUTINE = 5;
 }
 
+// Professional working status, from the time-use survey
+enum PwkStat {
+    // Not applicable (age < 16)
+    NA = 0;
+    // Full-time employee
+    EMPLOYEE_FT = 1;
+    // Part-time employee
+    EMPLOYEE_PT = 2;
+    // Unspecified employee
+    EMPLOYEE_UNSPEC = 3;
+    // Self-employed
+    SELF_EMPLOYED = 4;
+    // Unemployed (note this has a PWK_ prefix due to protobuf scoping rules)
```

Comment:
Proto3 is annoying; you can't use the same name in two different enums in the same package. That's silly, because most languages scope the names appropriately (`PwkStat::UNEMPLOYED` vs `NSSEC5::UNEMPLOYED`). But protobuf has to cater to the lowest common denominator of the languages it supports, and apparently C++ namespaces are weird.